client/app/components/sections/Topbar/Topbar.css

```diff
@@ -52,10 +53,26 @@
     font-size: var(--Menu_fontSizeSmall);
   }
 
+  & > * {
+    margin-right: var(--Topbar_itemSpacing);
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+}
+
+.topbarSpacer {
+  display: none;
+  @media (--Topbar_medium) {
```

Comment:
This will require a newline before the media query for Stylelint (might be hard to see with the menu errors atm).